sdk-workflows/src/main/java/io/dapr/workflows/client/DaprWorkflowClient.java

```diff
@@ -14,32 +14,50 @@
 package io.dapr.workflows.client;
 
 import com.microsoft.durabletask.DurableTaskClient;
+import com.microsoft.durabletask.DurableTaskGrpcClientBuilder;
 
-public class DaprWorkflowClient {
+import javax.annotation.Nullable;
+
+public class DaprWorkflowClient implements AutoCloseable {
 
   private DurableTaskClient innerClient;
 
   /**
    * Constructor for DaprWorkflowClient.
-   *
-   * @param innerClient DurableTaskClient instance
-   * @throws IllegalArgumentException if innerClient is null
    */
-  public DaprWorkflowClient(DurableTaskClient innerClient) throws IllegalArgumentException {
-    if (innerClient == null) {
-      throw new IllegalArgumentException("Inner client cannot be null");
-    } else {
-      this.innerClient = innerClient;
-    }
+  public DaprWorkflowClient() throws IllegalArgumentException {
+    DurableTaskClient innerClient = new DurableTaskGrpcClientBuilder().build();
```

Comment:
We didn't talk about this, but I think we should include port configuration as part of the setup for this client.

Check out [ActorClient](https://github.com/mindlessroman/java-sdk/blob/master/sdk-actors/src/main/java/io/dapr/actors/client/ActorClient.java#L87-L114) and [DurableTaskGrpcClientBuilder#grpcChannel](https://github.com/microsoft/durabletask-java/blob/main/client/src/main/java/com/microsoft/durabletask/DurableTaskGrpcClientBuilder.java#L40)

I think we can combine them into something like this:
```java
Channel channel = buildManagedChannel();
DurableTaskClient innerClient = new DurableTaskGrpcClientBuilder()
    .grpcChannel(channel)
    .build();
```

We would also need to shutdown the channel manually in our `close` method